mscore/qml/palettes/StyledCheckBox.qml

```diff
@@ -0,0 +1,43 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 MuseScore BVBA
```

Comment:
ZtHis too should rather be `//  Copyright (C) 2020 MuseScore BVBA and others`

Reply:
Done!